src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/InitializeHandler.cs

```diff
@@ -16,7 +16,12 @@ internal class InitializeHandler : IRequestHandler<InitializeParams, InitializeR
         {
             Capabilities = new ServerCapabilities
             {
-                // Placeholder until more capabilities are enabled
+                CompletionProvider = new CompletionOptions()
+                {
+                    AllCommitCharacters = new[] { " ", ".", ";", ">", "=" },
+                    ResolveProvider = false,
```

Comment:
Could you file an issue and put it into LSP Preview 3 for supporting completion resolution?

Reply:
Taylor - I started down this path, and then realized that your bug logging system differs greatly from ours. Can you log this? I can provide more details, but I think it's a bad path if you start to go down understanding what our trigger/commit characters are.

---
In reply to: [392326055](https://github.com/dotnet/aspnetcore-tooling/pull/1685#discussion_r392326055) [](ancestors = 392326055)

Reply:
@ToddGrun, I think what @NTaylorMullen is talking about is different from the issue you mentioned. @NTaylorMullen can you elaborate more on what we need to do for Completion resolution?

Reply:
Completion resolution is the content that gets shown in the description of the completion when you highlight the completion item:
![image](https://user-images.githubusercontent.com/2008729/76669760-4fdb9f80-654a-11ea-8b1a-09dba53a8c88.png)

I was expecting @ajaybhargavb to file the issue @ToddGrun no worries. It's not 100% necessary that the HTML language server understands the resolve request because it should just no-op. I'm more concerned with C# completion items having resolved information because it has a lot more vital info.

Reply:
Filed https://github.com/dotnet/aspnetcore/issues/19851